textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/utils/DataflowUtils.java

```diff
@@ -294,4 +295,25 @@ public static String getSpanString(Span span) {
         return payload;
     }
 
+    public static JSONArray getMetadataJSON(List<TableMetadata> list) {

```

Comment:
In next PR: add Jackson annotations to TableMetaData class and all classes it uses (schema, attribute, etc...).